### DISCUSSION TITLE
Can't find chat window even "GitHub Copilot chat private beta is already enabled on your account"

### DISCUSSION THREAD

**Discussion by anselmwang** [community/community#53508] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Currently when I click "join waitlist" button, GitHub notifies me "GitHub Copilot chat private beta is already enabled on your account".

In my VSCode, I have the "GitHub Copilot 1.83.41" (which released 2 days ago). 

From the command palette, I don't see how to open the chat window. 
![image](https://user-images.githubusercontent.com/7410579/233810722-be0a14eb-ff01-4dc3-be60-a4523642bae8.png)

How can I open the chat window? Thanks.

**Comment by albero94** [comment]
Same

**Comment by davecheney** [comment, marked as answer]
> "GitHub Copilot chat private beta is already enabled on your account".

This means you've signed up for the waitlist. When you are accepted you will receive an email.

**Reply by DanielLaberge** [reply]
That message is very misleading and causing a lot of confusion I bet.

**Reply by Nevrai** [reply]
Like Daniel said, this message is fairly misleading. I was looking for a setting to enable the chat mode.

In my humble opinion, the message should say that you’ve signed up for the wait list, not ‘chat private beta is already enabled on your account’.

**Reply by GanizaniSitara** [reply]
Isn't this theoretical now given it works ok on the main build?

**Comment by ZhengRui** [comment]
i received the email yesterday, but still no chat window or icon i can find :(

**Reply by davecheney** [reply]
You must install the latest vscode-insiders build. It has to be the latest, and you have to stay up to date every day.

**Reply by ZhengRui** [reply]
correct, I installed it in the vscode-insiders build and it works now, thanks !

**Reply by GanizaniSitara** [reply]
same just pulled todays copilot and todays insiders build. Email came yesterday

**Reply by mora145** [reply]
> You must install the latest vscode-insiders build. It has to be the latest, and you have to stay up to date every day.

Ty!

**Comment by oleg-alexandrov** [comment]
Indeed. Getting a new vscode insides build did the trick, even if the one I had was rather recent. Now the chat icon shows up.

**Reply by davecheney** [reply]
Yup, we're iterating rapidly so you need to apply the updates to -insiders and the -nightly build each day. Thank you.

**Comment by SimonGitter** [comment]
For reference Here is how the email looks like. I hope this helps someone :) 

-------------------------------------------------------------------
For Visual Studio Code:

Download and install [Visual Studio Code - Insiders](https://code.visualstudio.com/insiders/)
Already on the Insiders build? Make sure you are on the latest version!
In VS Code Insiders, go to the Extensions tab (Cmd+Shift+X) in the activity panel and search for and install the [GitHub Copilot Nightly extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly).
If you have not previously authorized VS Code Insiders in your GitHub account, follow the steps to do so, you will be prompted to sign in.
A new Chat icon appears in the activity panel. Click on it to see the conversation interface.
If that doesn't work, restart the editor and authorize GitHub Copilot to see the chat window.
For Visual Studio:

Ensure you're running Visual Studio 2022 17.5.1 or higher. If not, please update.
Download and install the [Visual Studio extension for GitHub Copilot - Preview](https://aka.ms/VSXGHCopilot)
Ensure you are logged in to Visual Studio using the same GitHub ID that you signed up for the waitlist using. If you are not signed with an authorized account you’ll be prompted to do so. If you are already logged in with your GitHub ID in your Visual Studio, please check you don’t need to re-enter your GitHub credentials. Here is the link with instructions to [Add your GitHub ID to your Visual Studio signin](https://learn.microsoft.com/visualstudio/ide/work-with-github-accounts?view=vs-2022).
Start Visual Studio. Once fully loaded, check that the GitHub Copilot chat window shows on your View menu. Then verify that “Ask Copilot” appears at the top of the right click menu in your code editor when a code file is open.
